docs/usage/exposureclasses.md

```diff
@@ -0,0 +1,92 @@
+# ExposureClasses
```

Comment:
Please add this document to the index (https://github.com/gardener/gardener/blob/master/docs/README.md)